src/modules/services_manager_service.rb

```diff
@@ -114,17 +97,19 @@ def enabled(service)
     # @param service [String] service name
     # @return [String]
     def state(service)
-      return nil unless exists?(service)
-      services[service][:state]
+      exists?(service) do
+        services[service].active_state
```

Comment:
You are right. I just though you had renamed them.

Reply:
On the other hand, why do we have `substate` and `sub_state` methods?

Reply:
In systemd the property is called "SubState". We are adding method names in SysmtedService using the property name but with snake case format. In SystemService I used `substate` instead of the name `sub_state` that comes from the systemd object, but only for a matter of personal preference.